presto-docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -23,47 +20,354 @@ connection properties as appropriate for your setup:
     connection-user=root
     connection-password=secret
 
-Multiple Oracle Servers
+By default, the Oracle connector uses connection pooling for performance
+improvement. The below configuration shows the typical default values. To update
+them, change the properties in the catalog configuration file:
+
+.. code-block:: properties
+
+    oracle.connection-pool.max-size=30
+    oracle.connection-pool.min-size=1
+    oracle.connection-pool.inactive-timeout=20m
+
+To disable connection pooling, update properties to include:
+
+.. code-block:: none
+
+    oracle.connection-pool.enabled=false
+
+.. note::
+
+    Include catalog files on all of your nodes, this requires you to restart.
+
+Multiple Oracle servers
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-You can have as many catalogs as you need, so if you have additional
-Oracle servers, simply add another properties file to ``etc/catalog``
-with a different name (making sure it ends in ``.properties``). For
-example, if you name the property file ``sales.properties``, Presto
-will create a catalog named ``sales`` using the configured connector.
+If you want to connect to multiple Oracle servers, configure another instance of
+the Oracle connector as a separate catalog.
+
+To add another Oracle catalog, create a new properties file. For example, if
+you name the property file ``sales.properties``, Presto creates a catalog named
+sales.
 
 Querying Oracle
 ---------------
 
-The Oracle connector provides a schema for every Oracle *database*.
-You can see the available Oracle databases by running ``SHOW SCHEMAS``::
+The Oracle connector provides a schema for every Oracle database. Use the below
+queries to test your setup.
+
+If you used a different name for your catalog properties file, use that catalog
+name instead of ``oracle``.
+
+.. note::
+    The user configuration, in the connection properties file, determines your
+    privileges in these schemas. If the Oracle user doesn't have access to a
+    table, you can't access it through Presto either.
+
+Run ``SHOW SCHEMAS`` to see the available Oracle databases::
 
     SHOW SCHEMAS FROM oracle;
 
-If you have a Oracle database named ``web``, you can view the tables
-in this database by running ``SHOW TABLES``::
+If you have an Oracle database named ``web``, run ``SHOW TABLES`` to see the
+tables it contains::
 
     SHOW TABLES FROM oracle.web;
 
-You can see a list of the columns in the ``clicks`` table in the ``web`` database
-using either of the following::
+To see a list of the columns in the ``clicks`` table in the ``web``
+database run either of the following::
 
     DESCRIBE oracle.web.clicks;
     SHOW COLUMNS FROM oracle.web.clicks;
 
-Finally, you can access the ``clicks`` table in the ``web`` database::
+To access the ``clicks`` table in the ``web`` database run::
```

Comment:
The original structure seems better. It looks strange to have "run" at the end of the sentence, since it's missing a comma and seems to be missing something. The correct way to write a sentence like this would be

> To access the clicks table in the web database, run the following: